evergreen.yml

```diff
@@ -154,6 +163,20 @@ functions:
             cargo set-version $RELEASE_VERSION
           fi
 
+  "sign windows":
+    - command: shell.exec
+      params:
+        working_dir: mongosql-odbc-driver/installer/msi
+        script: |
+          echo "${signing_token_odbc_driver}" > ./signing_auth_token
+          /usr/local/bin/notary-client.py \
+              --key-name "bi-connector" \
```

Comment:
Can we change the key name or is there some kind of signing variable tied to key-name, in which case we might want/need another key-name?

Reply:
Just saw this, yes the build team assigns the key-name and the auth-token associated with it to allow for the signing.  From their [script](https://github.com/10gen/notary-service/blob/316032a525af31a303e70e3fc6831cd7d5f4f332/notary/settings.py#L88) it looks like the same `pfx` file is used for signing of different projects, which if that's the case then getting a new one would be mainly to have a matching `odbc` keyname.  Seems like something we can create a ticket for later if we really wanted to.

Reply:
Yes, sounds good to me!